### ISSUE TITLE
Empty unused items button doesn't work

### ISSUE THREAD

**Issue by eort** [smathot/OpenSesame#476]
When going to `Unused Items` and clicking there on the  `Permanently delete unused items` button, an error message appears and the unused items are still present. Here is the error message:

~~~.python
# Oops, I did it again

An unexpected error occurred. Do you want to send a bug report? This is automatic, and prevents similar problems in the future.

[Send the error message](opensesame://event.bug_report_send){: .important-button}

Privacy:

- The error message (see below) may contain identifying information, such as your user name. If so, this information will not be used for any other purpose than fixing bugs.

Information:

- Operating System: linux2
- OpenSesame: 3.1.2b1
- Python: 2.7.6

Don't show me this message!

- If you don't want to see bug-report messages in the future, you can disable the bug_report extension under Menu → Tools → Plug-in and extension manager. This does not automagically solve the problem--it only suppresses the report!

Error message:

Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/libqtopensesame/widgets/unused_widget.py", line 85, in purge_unused
    self.overview_area.locked = True
AttributeError: 'unused_widget' object has no attribute 'overview_area'

~~~

Eduard

**Comment by eort** [comment]
Hm. After having updated to the newest version, this error disappeared. I suppose, there is no issue after all...

**Comment by smathot** [comment]
Yes, this was fixed in 852fd3248525957bda7735bd0f565e7890b3a2d1!
